templates/components/itilobject/timeline/form_followup.html.twig

```diff
@@ -297,6 +297,7 @@
                // set textarea content
                if (tasktinymce = tinymce.get("content_{{ rand }}")) {
                   tasktinymce.setContent(data.content);
+                  tasktinymce.fire('keyup');
```

Comment:
There are maybe other editors instances affected by this bug.
If usage of `setContent()` triggers the `Change` event, moving some logic in `editor.on('Change', function (e) { ... })` would be a better fix.

Reply:
If problem persists after merge of #15531, the best solution would be to move this call into `setRichTextEditorContent()` and add a comment to explain why it has been added.

Reply:
done